getting-started/vs-code-integration/new-project-wizard.md

```diff
@@ -25,21 +25,27 @@ To create a Telerik-enabled Blazor project:
 
 1. Type/Select `Telerik UI for Blazor Template Wizard: Launch` and press `Enter` to launch the extension.
 
-## Configure the Project
+## Creating the Project
 
 The **Create New Project** wizard provides two options to start you project:
 
 * **Start from Blank Project** - allows you to create a blank project that is pre-configured for the Progress® Telerik® UI for Blazor components. You can select and add sample pages to get started (a Grid with CRUD operations form a service, a Form, Charts).
 
-* **Start from a Template** - the wizard provides an Admin Dashboard Template - a fully functional sample app that showcases some of the most popular Telerik UI for Blazor components, such as Grid, TileLayout, Form, Charts, Gauges, etc. in a real world scenario.
+* **Start from a Template** - the wizard provides a set of predefined project templates:
+    * **CRUD, Form, Chart**&mdash;A small app that showcases the Telerik UI Data Grid, Chart, and forms validation. The Grid also uses a basic CRUD service which is a common pattern for data updates in real applications.
+    * **Dashboard**&mdash;A basic dashboard layout that uses the TileLayout component for customizable dashboards, and showcases several ways individual blocks (tiles) can fetch data.
+    * **Admin**&mdash;A small app that shows a dashboard for admin. The app uses some of the main Telerik UI components like the [Data Grid]({%slug grid-overview%}), [Chart]({%slug components/chart/overview%}), [TileLayout]({%slug tilelayout-overview%}), [Form]({%slug form-overview%}), and more.
 
->caption The wizard allows you to configure the following options:
+## Configuring the Project
+
+The wizard allows you to configure the following options:
 
 * Location and name for your app.
-* Your Telerik license (trial or commercial, depending on what you have).
-* A WebAssembly or Server-Side Blazor app.
+* Hosting model type - [WebApp]({%slug getting-started/web-app%}), [WebAssembly]({%slug getting-started/client-side%}), [Server-Side]({%slug getting-started/server-side%}) or [Hybrid Blazor MAUI]({%slug getting-started/hybrid-blazor%}) app.
+* Your Telerik license (trial or commercial, depending on what you have). 
+* Whether to use localization in the project or not.
+* Whether to use font icons in the project or not.
 * The .NET version you want to use.
```

Comment:
Can people choose a different .NET version? Does it depend on what they have installed on their system? If the latter, I would add a line saying `The wizard shows only the .NET versions available on your system.`

Reply:
The wizard provides a general .NET version configuration option regardless of what is installed on one's machine.

Reply:
If that's the case, I'd argue about the merit of a screenshot showing how people can choose from one possible option.